scorep/instrumenters/scorep_profile.py

```diff
@@ -1,9 +1,8 @@
 __all__ = ['ScorepProfile']
 
 import sys
-import inspect
-import os.path
-import scorep.instrumenters.base_instrumenter as base_instrumenter
+from scorep.instrumenters.base_instrumenter import get_module_name, get_file_name
```

Comment:
I am not really happy about this shortcut. I tend to always specify the full name of a function, including the module where it comes from. I know that I didn't do this for `base_instrumeter` but at least using `import scorep.instrumenters.base_instrumenter as base_instrumenter` its obvious that `get_module_name` and `get_file_name` are from a different module. This makes code a lot easier to read.

Reply:
I fully disagree. Especially here: Why is `get_module_name` even part of `base_instrumenter`? So a) I don't want/need to know at the callsite where this is coming from and b) it actually confuses me that I call such a function in `base_instrumenter`. Because `base_instrumenter.get_file_name(x)` looks like "get me the file name of the base instrumenter".   
So why does it make the code easier to read? I honestly can't see that.

So unless having only the name causes any kind of confusion I'd always use just the name.

I'm open to just using "your" way to avoid lengthy bikeshedding so take that into account and make a definite call how you would like entities that are not in the current file to be used. How much "absolute" it should be (at the callsite)

Reply:
As discussed in the telco, we stick to your way.